languages/common-lisp/exercises/concept/basic-lists/.docs/introduction.md

```diff
@@ -0,0 +1,85 @@
+### Lists in Common Lisp
```

Comment:
The introduction is currently quite long. We're aiming for [relatively concise and minimal introductions](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#docsintroductionmd) where the information provided should give the student just enough context to figure out the solution themselves.

What makes this introduction relatively big is that it uses a lot of code samples. Code examples should only be used to introduce new syntax (students should not need to search the web for examples of syntax). Many of the examples just show how specific functions can be used. In other words, they introduce new functions, not new syntax. As such, these examples should be omitted from the introduction (they're perfectly fine in an `after.md` document).

In many languages, the list functions are defined in a module or namespace. So for those languages, it is enough for the introduction to just explicitly mention that module which allows the student to find the relevant methods themselves (and if they can't, the hints should link to those methods). See the [F# lists exercise](https://github.com/exercism/v3/blob/master/languages/fsharp/exercises/concept/lists/.docs/introduction.md) for example.

Reply:
Looks like i had it backwards - i put lots of code examples in the introduction rather the instructions.

In Common Lisp all functions are in a single namespace: `COMMON-LISP` and the naming isn't always consistent (i.e. they don't all start with `LIST-`) for historical reasons (language was created by committee to produce a "common" lisp based on the multiple competing lisps at the time). So it is not always easy to find the function you want. The language specification is a GREAT resource as it is divided into sections such as 'characters', 'strings', 'sequences'. However it is NOT GREAT for a person new to the language.